lexicons/com/atproto/admin/createCommunicationTemplate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.createCommunicationTemplate",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Administrative action to create a new, re-usable communication (email for now) template.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["subject", "content", "name"],
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the template."
+            },
+            "content": {
```

Comment:
I'm sometimes a fan of indicating when a field has syntax, eg `contentMarkdown`.